src/api/app/controllers/webui/watch_items_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Webui::WatchItemsController < ApplicationController
```

Comment:
Going to leave some general feedback up here. You're missing

- Strong parameters
- Separation of concerns in your before filters
- Authorization